CHANGELOG.md

```diff
@@ -1,9 +1,15 @@
-# dbt_tiktok_ads v0.UPDATE.UPDATE
+# dbt_tiktok_ads v0.4.0
+[PR #11](https://github.com/fivetran/dbt_tiktok_ads/pull/11) includes the following changes:
 
- ## Under the Hood:
+## 🚨 Breaking Changes 🚨:
+- In the [July 2023 connector update for TikTok Ads](https://fivetran.com/docs/applications/tiktok-ads/changelog), we upgraded the connector to the v1.3 API. As a result the dependent fields and field names from the downstream staging models have changed.
+- None of the renamed fields were brought into the `dbt_tiktok_ads` package other than `age ` to `age_groups`, but it was subsequently removed due to an additional issue. We did need to update seed files with the renamed field names.
+- Removed JSON array fields (`action_categories`, `age_groups`, `interest_category` and `language`) from our models that were causing issues with BigQuery, which currently does not support JSON arrays in group by functions
```

Comment:
I would request we make this a bit more concise as to what is changing inform users what exactly was removed, and a short explainer as to why. For example, there is probably no reason to explain the renaming of age to age_groups if we are just removing it already. My suggestion would be something along the lines of the following:
```suggestion
- In the [July 2023 connector update for TikTok Ads](https://fivetran.com/docs/applications/tiktok-ads/changelog), the connector was updated to support the TikTok Ads v1.3 API. As a result breaking changes exist within the dependent [v0.4.0 dbt_tiktok_ads_source](https://github.com/fivetran/dbt_tiktok_ads_source/releases/tag/v0.4.0) release in addition to the following breaking changes within this package release:
| **model** | **removed_fields** |
| ----------| -------------------- |
| [tiktok_ads__ad_group_report](https://fivetran.github.io/dbt_tiktok_ads/#!/model/model.tiktok_ads.tiktok_ads__ad_group_report) | `action_categories`, `age`, `languages`, `interest_category` |
| [tiktok_ads__ad_report](https://fivetran.github.io/dbt_tiktok_ads/#!/model/model.tiktok_ads.tiktok_ads__ad_report) | `action_categories`, `age`, `languages`, `interest_category` |
| [tiktok_ads__url_report](https://fivetran.github.io/dbt_tiktok_ads/#!/model/model.tiktok_ads.tiktok_ads__url_report) | `action_categories`, `age`, `languages`, `interest_category` |

>**Note**: Some of the above fields were also removed due to complications with the BigQuery JSON datatype causing errors during compliation.
```

Reply:
Agreed, thanks for this suggestion.